docs/content/concepts/partitions-schedules-sensors/partitioning-assets.mdx

```diff
@@ -410,13 +366,56 @@ height={1618}
 
 ---
 
-## See it in action
+## Examples
 
 For more examples of partitions, check out the following in our [Hacker News example](https://github.com/dagster-io/dagster/tree/master/examples/project_fully_featured):
 
 - [Defining partitioned assets](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/assets/core/items.py)
 - [Defining a partitioned asset job and a schedule based on it](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/jobs.py)
 
+### Partitioned assets with partitioned I/O managers
+
+<Note>
+  <strong>Heads up!</strong> Familiarity with{" "}
+  <a href="/concepts/io-management/io-managers">I/O managers</a> is required for
+  this section.
+</Note>
+
+Asset functions can write data out to files, but they can also delegate the writing operation to an [I/O manager](/concepts/io-management/io-managers). Dagster's [built-in I/O managers](/concepts/io-management/io-managers#built-in-io-managers) support handling partitioned assets, but you can also [write your own I/O manager](/concepts/io-management/io-managers#handling-partitioned-assets) if you want additional customization.
+
+For example, this example demonstrates how to define an asset that relies on an I/O manager to store its output:
```

Comment:
```suggestion
For example, the following demonstrates how to define an asset that relies on an I/O manager to store its output:
```

Reply:
I am wondering if this previous line is a little confusing. The idea of the example seems to be to demonstrate how the default I/O manager uses partitions to write data to a subfolder? But the previous sentence just mentions the example shows how to write an asset (which I don't think is the intent of this section.)